content/kubernetes/release-notes/7-2-4-releases/7-2-4-2.md

```diff
@@ -116,12 +114,15 @@ Any distribution not listed below is not supported for production workloads.
 
 The limitations below are specific to the Redis Enterprise for Kubernetes product. To view limitations for the supported Redis Enterprise Software version, see the [7.2.4 release notes]({{<relref "/rs/release-notes/rs-7-2-4-releases#known-limitations">}}).
 
-{{<warning>}}In version 7.2.4, old module versions and manually uploaded modules are not persisted. If databases are not upgraded after cluster upgrade, and require cluster recovery afterwards, you'll need to contact Redis support. This issue will be fixed in the next maintenance release by moving the stored location of the modules.{{</warning>}}
 
 ### New limitations
 
+- **REDBs created with version 6.2.10-4 or before will automatically upgrade modules when the cluster is upgraded (RED-106834)**
+
 - **Cannot upgrade or recover clusters if there are databases with manually uploaded modules (RED-107711 RED-106872)** Contact support.
 
+{{<warning>}}In version 7.2.4, old module versions and manually uploaded modules are not persisted. If databases are not upgraded after cluster upgrade, and require cluster recovery afterwards, you'll need to contact Redis support. This issue will be fixed in the next maintenance release by moving the stored location of the modules.{{</warning>}}
```

Comment:
I think we should not promise that this will be fixed in the next maint release.

Reply:
I don't have a problem with this. Already fixed in RS and CP'd into the Sep 13 RS maintenance release, as planned.

Reply:
ok, but keep in mind that wasn't tested via us on K8s only in theory (though I don't think this should not work).